ccd-definition/CaseField/CareSupervision/manageOrders/manageOrdersFields.json

```diff
@@ -166,6 +166,35 @@
     "SecurityClassification": "PUBLIC",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "manageOrdersC43Orders",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "C43OrderType",
+    "Label": "Select orders to issue",
+    "SecurityClassification": "PUBLIC",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "manageOrdersRecitalsAndPreambles",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "FieldType": "TextArea",
+    "Label": "Add recitals or preamble",
```

Comment:
<img width="547" alt="Screenshot 2021-07-01 at 16 28 19" src="https://user-images.githubusercontent.com/30896346/124150312-5bc5da80-da89-11eb-8805-7a471164286d.png">

Great attention to detail @SooryaSu . Good catch with entering the correct spelling